dask_jobqueue/lsf.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import absolute_import, division, print_function
+
+import logging
+import math
+import os
+
+import dask
+
+from .core import JobQueueCluster, docstrings
+
+logger = logging.getLogger(__name__)
+
+
+class LSFCluster(JobQueueCluster):
+    __doc__ = docstrings.with_indents(""" Launch Dask on a LSF cluster
+
+    Parameters
+    ----------
+    queue : str
+        Destination queue for each worker job. Passed to `#BSUB -q` option.
+    project : str
+        Accounting string associated with each worker job. Passed to
+        `#BSUB -P` option.
+    walltime : str
+        Walltime for each worker job HH:MM. Passed to `#BSUB -W` option.
+    job_cpu : int
+        Number of cpus. Passed to `#BSUB -n` option.
+    job_mem : str
+        Request memory. Passed to `#BSUB -M` option.
+    job_extra : list
+        List of other LSF options, for example -e. Each option witll be
+        prepended with the #LSF prefix.
+    %(JobQueueCluster.parameters)s
+
+    Examples
+    --------
+    >>> from dask_jobqueue import LSFCluster
+    >>> cluster = LSFcluster(queue='general', project='DaskonLSF')
+    >>> cluster.start_workers(10)  # this may take a few seconds to launch
+
+    >>> from dask.distributed import Client
+    >>> client = Client(cluster)
+
+    This also works with adaptive clusters.  This automatically launches and
+    kill workers based on load.
+
+    >>> cluster.adapt()
+    """, 4)
+
+    # Override class variables
+    submit_command = 'bsub'
```

Comment:
From what I understand from your last comment, the `submit_command` should be `bsub <` instead of just `bsub`.

Reply:
@raybellwaves any response to this comment?